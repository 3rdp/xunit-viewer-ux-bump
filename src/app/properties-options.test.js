import React from 'react'
import PropertiesOptions from './properties-options'
import renderer from 'react-test-renderer'

test('renders properties options', () => {
  const tree = renderer.create(<PropertiesOptions count={100} total={999} />).toJSON()
  expect(tree).toMatchSnapshot()
})
