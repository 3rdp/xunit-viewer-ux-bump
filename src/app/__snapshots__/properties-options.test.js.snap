// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properties options 1`] = `
<div
  className="options card is-inactive"
>
  <header
    className="card-header"
  >
    <div
      className="field options-search"
    >
      <div
        className="control"
      >
        <input
          className="input"
          onChange={[Function]}
          placeholder="Properties"
          type="text"
        />
      </div>
    </div>
    <button
      className="button card-header-icon"
      onClick={[Function]}
    >
      <div
        className="options-inputs"
      >
        <div
          className="options-total"
        >
          <b>
            100
          </b>
          <span>
            /
            999
          </span>
        </div>
      </div>
      <span
        className="icon"
      >
        <i
          className="fas fa-angle-down"
        />
      </span>
    </button>
  </header>
  <div
    className="card-content"
  />
</div>
`;
